<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="PythonProjectDevelop.cafe.apps.menu.tests.TeaManagerTest-20230321131310" tests="1" file="PythonProjectDevelop/cafe/apps/menu/tests.py" time="0.004" timestamp="2023-03-21T13:13:10" failures="1" errors="0" skipped="0">
	<testcase classname="PythonProjectDevelop.cafe.apps.menu.tests.TeaManagerTest" name="test_count_each_kind" time="0.004" timestamp="2023-03-21T13:13:10" file="menu/tests.py" line="25">
		<failure type="AssertionError" message="{'chinese': 3, 'english': 2, 'japanese': 1} != {'english': 2, 'chinese': 3, 'japanese': 3}
- {'chinese': 3, 'english': 2, 'japanese': 1}
?                                          ^

+ {'chinese': 3, 'english': 2, 'japanese': 3}
?                                          ^
"><![CDATA[Traceback (most recent call last):
  File "/Users/iced_me/Developer/Python/PythonProjectDevelop/cafe/apps/menu/tests.py", line 27, in test_count_each_kind
    self.assertEqual(result, dict(english=2, chinese=3, japanese=3))
AssertionError: {'chinese': 3, 'english': 2, 'japanese': 1} != {'english': 2, 'chinese': 3, 'japanese': 3}
- {'chinese': 3, 'english': 2, 'japanese': 1}
?                                          ^

+ {'chinese': 3, 'english': 2, 'japanese': 3}
?                                          ^

]]></failure>
	</testcase>
</testsuite>
